installation-bundle/src/Database/Version410Update.php

```diff
@@ -40,7 +40,7 @@ public function shouldRun(): bool
     {
         $schemaManager = $this->connection->getSchemaManager();
 
-        if (!$schemaManager->tablesExist(['tl_user'])) {
+        if (!$schemaManager->tablesExist(['tl_user', 'tl_image_size'])) {
```

Comment:
```suggestion
        if (!$schemaManager->tablesExist(['tl_user', 'tl_user_group', 'tl_image_size'])) {
```
Should we add tl_user_group too?

Reply:
added in 012fce558a729cc28ab32873b237dbac7cf37374